README.md

```diff
@@ -76,10 +76,7 @@ We are using [Cypress](https://docs.cypress.io/guides/overview/why-cypress) and
 1. Navigate to the base of this repo and run `npx cypress run` for tests to execute in the command line. To execute tests in Cypress' UI window, run `npx cypress open` and select E2E testing.
 1. We have added functionality that will print an error table in the command line on test execution. This table includes the name and description of the error along with a `helpUrl` containing a URL where more information on how to fix that particular error can be found. Additionally, you can visit the official [axe-core rules](https://github.com/dequelabs/axe-core/blob/develop/doc/rule-descriptions.md) github to view the different axe-core rule descriptions.
 1. Cypress captures snapshots at the time of test execution, these can be viewed by navigating to the `cypress/screenshots` folder.
-### **Additional Notes on Cypress Testing for Local Development
-
-- During local development, it is important to restart your server if you're making code or package changes while http://localhost:8080 is still running. This will ensure the Cypress tests are running and passing correctly in your local environment.
 
-- If you are running Cypress tests in Docker and/or on a virtual machine, there are [additional dependencies](https://docs.cypress.io/guides/continuous-integration/introduction#Dependencies) you will need to install depending on what type of environment you have.
+### **Additional Notes on Cypress Testing for Local Development
 
-- If you are running Docker on your machine and get an error that `Cypress is not installed` and that it cannot find the cypress file at `/root/.cache/Cypress/12.17.1/Cypress`, this could be a permissions issue and you would run `node_modules/.bin/cypress install` inside of the Docker container to install Cypress.
```

Comment:
We may want to keep this bit of information for local development since the Docker container throws an error dependent on permissions. I know for my system this error and fix would be handy to know about. If you don't think this will ever be run locally then we probably wouldn't need it.

Reply:
@SMakaiTakori : Sounds good! I'll add the line back in. With that said, folks shouldn't get the error anymore, since the Dockerfile contains `node_modules/.bin/cypress install` so that permissions are always set correctly to run within the container. See the line at https://github.com/bloom-works/guides-template/pull/61/files#diff-dd2c0eb6ea5cfc6c4bd4eac30934e2d5746747af48fef6da689e85b752f39557R34

Reply:
Added back!